tensorflow/lite/micro/tools/make/third_party_downloads.inc

```diff
@@ -40,5 +40,3 @@ EMBARC_MLI_PRE_COMPILED_URL := "https://github.com/foss-for-synopsys-dwc-arc-pro
 EMBARC_MLI_PRE_COMPILED_MD5 := "173990c2dde4efef6a2c95b92d1f0244"
 
 # Skip md5sum-check since ethos-u-core-driver download link is non-deterministic, see  https://github.com/google/gitiles/issues/84
-ETHOSU_URL := "https://review.mlplatform.org/plugins/gitiles/ml/ethos-u/ethos-u-core-driver/+archive/24455eedb9e8939f8a28ca0101a6f2d171e1b2f9.tar.gz"
```

Comment:
Did you have a particular reason for migrating from using a download link to using git clone? The URL approach would continue to work by simply updating the link to: 
```
https://review.mlplatform.org/plugins/gitiles/ml/ethos-u/ethos-u-core-driver/+archive/9622608a5cc318c0933bcce720b59737d03bfb6f.tar.gz
```

Regardless of HTTP or git download, it is important that we version based on the SHA, not the tag. Tags can be moved, which can result in our scripts breaking (this has happened before). While SHAs can technically be erased too, that is significantly less likely.

Reply:
> Did you have a particular reason for migrating from using a download link to using git clone?

I was just following the existing pattern for the other Ethos platform package, which is also a git repo. It's also easier to tell which specific version of the driver is being used from a tag or SHA than from a download URL or even the downloaded package. Finally, probably only important for development, but if you want to check out a different commit for testing something, you can't do that with a downloaded package.

I've switched to commit SHA and retested the build, and retested on hardware.